dm/master/dm-master.toml

```diff
@@ -22,6 +22,15 @@ advertise-peer-urls = "http://127.0.0.1:8291"
 # initial cluster configuration for bootstrapping, e,g. dm-master=http://127.0.0.1:8291
 initial-cluster = "dm-master=http://127.0.0.1:8291"
 
+# Initial cluster state ("new" or "existing").
+# Set to "new" for all members present during initial static or DNS bootstrapping.
+# If this option is set to "existing", DM-master will attempt to join the existing cluster.
+# If the wrong value is set, DM-master will attempt to start but fail safely.
+initial-cluster-state = "new"
```

Comment:
when dm-master restart, need update it to "existing"?

Reply:
I also have doubts about the behavior of first start, join start,  and also the restart later, I think we better have a complete discussion.

Reply:
removed this section in sample config, and add some comments in https://github.com/pingcap/dm/pull/350/commits/dcda7b23a1dad59bbdc8dd3d69e12d9cb89cc623.